hydjet2.spec

```diff
@@ -0,0 +1,20 @@
+### RPM external hydjet2 2.x.y 
+
+Source: http://cern.ch/lokhtin/hydjet++/%{n}.tar.gz
```

Comment:
the URL `http://cern.ch/lokhtin/hydjet++/hydjet2.tar.gz` is very generic and our packaging system is not going to redownload it when you need to update the version. Can you please provide separate tar files for each version?

Reply:
At the author's site is just one version for the new HYDJET++ - the actual one. This naming policy implies to move all outdated version to other links/tarballs. The author is the member of CMS HI, since very beginning and he was updating PYQUEN and HYDJETs focusing primarily on the interests of the CMS. As far as I understand, during compiling of each CMSSW version, the external archive downloaded from the source website, so if author will update the version, it will be available for the next CMSSW version. It's ok for CMS HI group, we always need the more actual version on top.

Reply:
Or You mean difficulties related to the backport for the new version?

Reply:
For each version of a package there should be a unique url. If author just keeps on updating http://cern.ch/lokhtin/hydjet++/hydjet2.tar.gz (pointing to latest version) then there is no guarantee that two cmssw builds will have the same contents. I have no issues with http://cern.ch/lokhtin/hydjet++/hydjet2.tar.gz pointing to latest version but there should link for specific versions too e.g. http://cern.ch/lokhtin/hydjet++/hydjet2-%{version}.tar.gz and this should be used in our spec files.

Reply:
Note that cmsBuild ( our packaging system) also caches the contents of each source URL. So cmsBuild will always use the previously cached contents if url itself is not changed.

Reply:
I see, thanks. I will ask for this changes.